crypto/err/err_all_legacy.c

```diff
@@ -43,13 +43,20 @@
 #include "crypto/x509err.h"
 #include "crypto/x509v3err.h"
 
+#ifdef OPENSSL_NO_ERR
```

Comment:
There is wrong formatting of the preprocessor directives throughout the file. As the file is enclosed in `#ifdef` there should be a space added here and above.